howto/regex.po

```diff
@@ -32,160 +32,250 @@ msgstr ""
+"qui sont ensuite exécuté par un moteur de correspondance écrit en C. Pour une "
+"usage avancé, il peut être nécessaire de faire attention à comment le moteur "
+"exécutera la RE fournie, et écrire la RE d'une telle manière qu'elle produise "
+"un **bytecode** s'exécutant plus rapidement. L'optimisation n'est pas traitée "
+"dans ce document, parce qu'elle nécessite d'avoir une bonne compréhension des "
+"mécanismes internes du moteur de correspondance."
 
 #: ../Doc/howto/regex.rst:42
 msgid ""
-"The regular expression language is relatively small and restricted, so not "
-"all possible string processing tasks can be done using regular expressions.  "
-"There are also tasks that *can* be done with regular expressions, but the "
-"expressions turn out to be very complicated.  In these cases, you may be "
-"better off writing Python code to do the processing; while Python code will "
-"be slower than an elaborate regular expression, it will also probably be "
-"more understandable."
-msgstr ""
+"The regular expression language is relatively small and restricted, so not all "
+"possible string processing tasks can be done using regular expressions.  There "
+"are also tasks that *can* be done with regular expressions, but the expressions "
+"turn out to be very complicated.  In these cases, you may be better off writing "
+"Python code to do the processing; while Python code will be slower than an "
+"elaborate regular expression, it will also probably be more understandable."
+msgstr ""
+"Le langage d'expressions régulières est relativement petit et restreint, donc "
+"toutes les tâches de manipulation de chaînes de caractères ne peuvent pas être "
+"réalisées à l'aide d'expressions régulières. Il y a aussi des tâches qui "
+"*peuvent* être réalisées à l'aide d'expressions régulières, mais qui ont "
+"tendance à produire des expressions régulières très compliquées. Dans ces cas, "
+"il vous sera plus utile d'écrire du code Python pour réaliser le traitement; "
+"même si le code Python sera plus lent qu'une expression régulière élaborée, il "
+"sera probablement plus compréhensible."
 
 #: ../Doc/howto/regex.rst:51
 msgid "Simple Patterns"
-msgstr ""
+msgstr "Motifs simples"
 
 #: ../Doc/howto/regex.rst:53
 msgid ""
-"We'll start by learning about the simplest possible regular expressions.  "
-"Since regular expressions are used to operate on strings, we'll begin with "
-"the most common task: matching characters."
+"We'll start by learning about the simplest possible regular expressions.  Since "
+"regular expressions are used to operate on strings, we'll begin with the most "
+"common task: matching characters."
 msgstr ""
+"Nous commencerons par étudier les expressions régulières les plus simples. Dans "
+"le mesure où les expressions régulières sont utilisées pour manipuler des "
+"chaînes de caractères, nous commencerons par les tâches les plus communes: "
+"caractères ressemblants."
```

Comment:
"ressemblants" is badly choosen here, semantically we're just trying to express, by matching characters, that "toto" matches "toto".

Reply:
Done